content/blog/2017/2017-04-10-Jenkins-has-upgraded-to-Java-8.adoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Jenkins Now Requires Java 8 To Run Starting With 2.54"
+tags:
+- java8
+- upgrade
+author: batmat
+---
+
+:imagesdir: /images/post-images/2017-04-java8
+
+image:lambda.png[role="right"] As link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[announced some weeks ago], the necessary changes have now been merged into Jenkins codebase.
```

Comment:
> the necessary changes have now been merged into Jenkins codebase

This seems to be missing an explanation of the changes necessary for _what_. Sure, it's in the title, but still.